modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -149,3 +157,38 @@
                :parameters [{:type :date/all-options
                              :target [:dimension [:template-tag "date"]]
                              :value "past30years"}]}))))))
+
+(deftest redshift-types-test
+  (mt/test-driver
+    :redshift
+    (testing "Redshift specific types should be synced correctly"
+      (let [db-details   (tx/dbdef->connection-details :redshift)
+            tbl-nm       "redshift_specific_types"
+            qual-tbl-nm  (str rstest/session-schema-name "." tbl-nm)
+            view-nm      "late_binding_view"
+            qual-view-nm (str rstest/session-schema-name "." view-nm)]
+        (mt/with-temp
+          Database
+          [database {:engine :redshift, :details db-details}]
+          ;; create a table with a CHARACTER VARYING and a NUMERIC column, and a late bound view that selects from it
+          (#'rstest/execute!
+            (str "DROP TABLE IF EXISTS %1$s;%n"
+                 "CREATE TABLE %1$s(weird_varchar CHARACTER VARYING(50), numeric_col NUMERIC(10,2));%n"
+                 "CREATE OR REPLACE VIEW %2$s AS SELECT * FROM %1$s WITH NO SCHEMA BINDING;")
+            qual-tbl-nm
+            qual-view-nm)
+          ;; sync the schema again to pick up the new view (and table, though we aren't checking that)
+          (sync/sync-database! database)
+          (is
+            (contains?
+              (db/select-field :name Table :db_id (u/get-id database)) ; the new view should have been synced
+              view-nm))
```

Comment:
This should be indented one space -- see https://guide.clojure.style/#one-space-indent vs https://guide.clojure.style/#body-indentation to understand when you should use which.

What editor are you using? A good editor should indent things correctly automatically for you

Reply:
Cursive.  I'm sure it can be made to do all of this.  Still learning all the options and configuration, but I'll get it there.